Testing the radicle-node package fails

Testing the radicle-node package is failing, and its not being picked up when running `cargo test --all`.  Could we have the tests included in our PR tests?

```
cargo test --package radicle-node
error[E0433]: failed to resolve: could not find `test` in `crypto`
 --> radicle/src/test/arbitrary.rs:6:13
...
```

--- a/radicle-node/Cargo.toml
+++ b/radicle-node/Cargo.toml
@@ -34,5 +34,6 @@ features = ["sql"]
 
 [dev-dependencies]
 radicle = { path = "../radicle", version = "*", features = ["test"] }
+radicle-crypto = { path = "../radicle-crypto", version = "*", features = ["test"] }
 quickcheck = { version = "1", default-features = false }
 quickcheck_macros = { version = "1", default-features = false }

